app/Repositories/Eloquents/FollowEloquentRepository.php

```diff
@@ -31,7 +31,7 @@ public function store($data = [])
         }
     }
 
-     public function destroy($data){
+    public function destroy($data){
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/FollowEloquentRepository.php</br>Opening brace should be on a new line